lib/adapters/leveldb/index.js

```
@@ -10,7 +10,16 @@ function requireLeveldown() {
   // de-optimalization
   try {
     originalLeveldown = require('leveldown');
-  } catch (e) {}
+  } catch (err) {
+      if (err && err.message && err.message.match('Module version mismatch')) {
+          throw new Error([
+            err.message,
+            'This generally implies that leveldown was built with a different',
+            'version of node than that which is running now.  You may try',
+            'fully removing and reinstalling PouchDB or leveldown to resolve.'
+          ].join(' '));
+      }
```

again today, i had a [leveldown error swallowed](https://gist.github.com/cdaringe/e25beed841c756c4e7dd).